CONTRIBUTING.rst

```diff
@@ -0,0 +1,80 @@
+How to contribute
+=================
+
+All contributions are greatly appreciated!
+
+
+Setting up your development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-   Fork pytest-xprocess to your GitHub account by clicking the `Fork`_ button.
+-   `Clone`_ the main repository (not your fork) to your local machine.
+
+    .. code-block:: text
+
+        $ git clone https://github.com/pytest-dev/pytest-xprocess
+        $ cd pytest-xprocess
+
+-   Add your fork as a remote to push your contributions.Replace
+    ``{username}`` with your username.
+
+    .. code-block:: text
+
+        git remote add fork https://github.com/{username}/pytest-xprocess
+
+-   Create a new virtualenv.
+
+    .. code-block:: text
+
+        $ python3 -m venv env
+        $ . env/bin/activate
+
+-   Install pytest-xprocess in editable mode with development dependencies.
+
+    .. code-block:: text
+
+        $ pip install -e . -r requirements/dev.txt
+
+-   Finally, install the pre-commit hooks and you are
+    are all set to start coding.
+
+    .. code-block:: text
+
+        $ pre-commit install
```

Comment:
Hmmm following the above change, we should add `pre-commit` to `tox.ini` in the normal `[testenv]` for convenience as well.

Reply:
pre-commit added to [testenv:dev]